api/api.proto

```diff
@@ -890,6 +890,26 @@ message CapacityUsageInfo {
   int64 total_bytes = 3;
 }
 
+message VolumeUsage {
+   // VolumeID is the id for the volume
```

Comment:
fyi, there is no need to add the "golang" name in the comment because on gRPC the clients are for different languages. A lot of the users use python.

Instead you can make it look like this:

```proto
  // Id for the volume
  string volume_id = 1;
```

Example: https://libopenstorage.github.io/w/release-6.4.generated-api.html#sdkvolumeattachoptions

Reply:
Done